workspace/TS100/Core/BSP/Pine64/FreeRTOSConfig.h

```diff
@@ -1,21 +1,18 @@
 #ifndef FREERTOS_CONFIG_H
 #define FREERTOS_CONFIG_H
 /* Ensure stdint is only used by the compiler, and not the assembler. */
-#if defined(__ICCARM__) || defined(__CC_ARM) || defined(__GNUC__)
```

Comment:
This comment appears to be related to that #if you removed... might remove the comment too, or state what exactly is happening?